apps/backend/app.py

```diff
@@ -43,11 +48,22 @@
 firebaseDb = firestore.client()
 firebaseBucket = storage.bucket("gladosbase.appspot.com")
 
+
+#MongoDB Objects
+mongoClient = pymongo.MongoClient('glados-mongodb', 27017, serverSelectionTimeoutMS=1000)
+print(mongoClient.server_info)
```

Comment:
if we still want this print, can you make it also say "MongoDB connection info" or similar so that if it ever prints blanks it can still be tracked down?

Reply:
print was for testing purposes only, can be removed without issue